lib/mini_racer/version.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 module MiniRacer
-  VERSION = "0.8.0"
-  LIBV8_NODE_VERSION = "~> 18.16.0.0"
+  VERSION = "0.9.0"
+  LIBV8_NODE_VERSION = "~> 19.9.0.0"
```

Comment:
maybe we should skip non-LTS ?

Reply:
You can find some more context to the plan we made a while ago here, @mathieujobin: https://github.com/rubyjs/mini_racer/issues/277

Reply:
Thanks, that indeed explains a lot.

Makes a lot of sense.

looking forward testing 0.10.0 with node20